flask_app/tasks/replications.py

```diff
@@ -214,15 +208,41 @@ def _reconfigure_replica(replica):
                         "match_mapping_type": "string",
                         "mapping": {
                             "type": "text",
-                            "fields": {
-                                "keyword": {
-                                    "ignore_above": str(_ES_MAX_STRING_LENGTH),
-                                    "type":  "keyword"}}}}}]})
+                            "ignore_above": str(_ES_MAX_STRING_LENGTH),
+                            }}}]})
+
+
+
+def _get_properties_definitions():
+    returned = {}
+
+    for datetime_field in ['start_time', 'end_time']:
+        returned[datetime_field] = {
+            "type":   "date",
+            "format": "epoch_second"
+        }
+
+    for keyword_field in [
```

Comment:
i think we should also add:  session_metadata.Build url, session_metadata.slash_infinidat Version, test_metadata.backslash_link, user_email

Reply:
Only user_email is relevant here... the rest are Infinidat-specific...

Reply:
@maormarcus updated and pushed